.github/workflows/publish.yml

```diff
@@ -0,0 +1,50 @@
+name: Push on main
+on:
+  workflow_dispatch:
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+          - uses: actions/checkout@v3
+            with:
+              fetch-depth: 0
```

Comment:
don't need this since we're not analyzing the code history

Reply:
removed